utilities/infra.py

```
@@ -1146,6 +1146,13 @@ def get_oc_console_cli_download_link(admin_client: DynamicClient) -> str:
     return all_links[0]
 
 
+@retry(wait_timeout=120, sleep=10, exceptions_dict={requests.exceptions.ConnectionError: []})
+def _download_with_retry(url: str) -> requests.Response:
+    response = requests.get(url, verify=False, stream=True)
```

Ruff (0.15.17)

[error] Probable use of `requests` call without timeout

(S113)